Packs/Packetsled/Integrations/Packetsled/Packetsled.py

```diff
@@ -0,0 +1,500 @@
+        result = validate_response(response)
+        return {
+            'probes': result['rows'],
+            'dbs': list(map(lambda x: 'probe_{envid}_{probe}'.format(**x), result['rows']))
+        }
+
+
+def make_timerange(dargs):
+    if 'stop_time' in dargs:
+        tnow = getTime(dargs['stop_time'])
+    else:
+        tnow = time.mktime(datetime.now().timetuple())
+
+    if 'start_time' in dargs:
+        tmin = getTime(dargs['start_time'])
+    else:
+        lastRun = demisto.getLastRun() and demisto.getLastRun()["time"]
+        if len(lastRun) != 0:
+            tmin = getTime(lastRun)
+        else:
+            tmin = tnow - 1 * 60 * 60
+
+    return tmin, tnow
+
+
+def make_query(dargs):
+    tmin, tnow = make_timerange(dargs)
+
+    search = {'time': {'=': {'scalars': [], 'ranges': [{'v1': tmin, 'v2': tnow}]}}}
+
+    if 'entity' in dargs:
+        entity = coalesceToArray(dargs['entity'])
+        search['ip'] = {
+            '=': {
+                'scalars': map(lambda x: {'v1': ip_to_number(x)}, entity),
+                'ranges': []
+            }
+        }
+
+    if 'protocol' in dargs:
+        protocol = coalesceToArray(dargs['protocol'])
+        search['proto'] = {
+            '=': {
+                'scalars': map(lambda x: {'v1': proto_map[x]}, protocol),
+                'ranges': []
+            }
+        }
+
+    if 'family' in dargs:
+        family = coalesceToArray(dargs['family'])
+        search['family'] = {
+            '=': {
+                'scalars': map(lambda x: {'v1': family_map[x]}, family),
+                'ranges': []
+            }
+        }
+
+    if 'geo' in dargs:
+        geo = coalesceToArray(dargs['geo'])
+        search['geo'] = {
+            '=': {
+                'scalars': map(lambda x: {'v1': x}, geo),
+                'ranges': []
+            }
+        }
+
+    if 'port' in dargs:
+        port = coalesceToArray(dargs['port'])
+        search['port'] = {
+            '=': {
+                'scalars': map(lambda x: {'v1': x}, port),
+                'ranges': []
+            }
+        }
+
+    return search
+
+
+def main():
+    auth_token = ''
+    apiserver = demisto.params()['ApiServer']
+    username = demisto.params()['credentials']['identifier']
+    password = demisto.params()['credentials']['password']
+
+    response = requests.post(urljoin(apiserver, '/api/login'), data={'email': username, 'password': password},
+                             headers={'cache-control': 'no-cache'}, verify=VERIFY)
+
+    if response.status_code == 200:
+        result = response.json()
+        if result["status"] == 1:
+            auth_token = result["token"]
+
+    if not auth_token:
+        raise ValueError('Authorization failed')
+
+    if demisto.command() == 'test-module':
+        demisto.results('ok')
+        sys.exit(0)
+
+    if demisto.command() == 'packetsled-sensors':
```

Comment:
```suggestion
    elif demisto.command() == 'packetsled-sensors':
```
Also to all checks for commands below.